src/main/index.ts

```diff
@@ -77,6 +77,7 @@ app.on("ready", async () => {
 
   // run proxy
   try {
+    // eslint-disable-next-line unused-imports/no-unused-vars-ts
     proxyServer = LensProxy.create(proxyPort, clusterManager);
```

Comment:
I think `let proxyServer` (at the beginning of this file) can be removed and this line can be just
```
LensProxy.create(proxyPort, clusterManager)
```

Reply:
I was concerned that the GC might consider this to be "unreachable" because of that.... but I have no real basis for that.